### DISCUSSION TITLE
Continuing a Content Warning Module with Drew Brasher - Orchard Core Pair Programming by Lombiq

### DISCUSSION THREAD

**Discussion by Piedone** [OrchardCMS/OrchardCore#16125] [Show and tell]
Join this session of Orchard Core Pair Programming by Lombiq! In these, we do an hour of pair programming with an Orchard Core community member about a project of theirs. We learn together a lot, share best practices, and write some good code. All this is live, and you can join us with your questions! Check it out here: https://www.youtube.com/watch?v=SmvLpAfyEHI. See you at 5 PM UTC on the 28th!

@DrewBrasher will join us again with a Content Warning Module project that she started in the previous stream (https://www.youtube.com/watch?v=ECEbkLSOqLQ and source: https://github.com/DrewBrasher/OrchardCoreModules). We implemented a shortcode provider last time; now, we'll develop a content part to display a content warning for a whole content item.

Your host and the "navigator" of the pair programming session will be me.

Would you like to be our guest? Just comment here!

**Comment by Piedone** [comment]
We're starting in five minutes! Tune in here: https://www.youtube.com/watch?v=SmvLpAfyEHI

**Comment by hishamco** [comment]
I miss the stream today, I will check the recording verion
